jaeger-core/src/main/java/com/uber/jaeger/samplers/ProbabilisticSampler.java

```diff
@@ -26,32 +26,38 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
-import lombok.ToString;
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Value;
 
-@SuppressWarnings("EqualsHashCode")
-@ToString
+@Value
+@Getter(AccessLevel.NONE)
+@EqualsAndHashCode(of = {"samplingRate"})
 public class ProbabilisticSampler implements Sampler {
   public static final String TYPE = "probabilistic";
 
-  private final long positiveSamplingBoundary;
-  private final long negativeSamplingBoundary;
-  private final double samplingRate;
-  private final Map<String, Object> tags;
+  @Getter
+  double samplingRate;
+
+  long positiveSamplingBoundary;
+  long negativeSamplingBoundary;
```

Comment:
I still don't understand why we're not declaring these private final. I don't care if lombok makes them final automatically or not, it improves readability/understanding of the class by stating this explicitly.